app/com/arpnetworking/metrics/portal/reports/ReportingJobRepository.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.arpnetworking.metrics.portal.reports;
+
+// TODO(cwbriones): Decouple from ebean model and expose via the internal model.
+import models.ebean.ReportingJob;
+
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * A repository for storing and retrieving instances of <code>ReportingJob</code>.
```

Comment:
Ideally, we should use `{@link ...}` or `{@code ...}`. The use of `<code>...</code>` is generally frowned upon. Unfortunately, this project has not been updated.